src/tutorial/src/step-1/description.md

```diff
@@ -1,26 +1,26 @@
-# Getting Started {#getting-started}
+# Come Iniziare {#getting-started}
 
-Welcome to the Vue tutorial!
+Benvenuti nel tutorial su Vue!
```

Comment:
In genere usiamo il singolare. mi sa che dovremo aggiungerlo come regola scritta

Reply:
Questa cosa effettivamente va scritta da qualche parte